docker-qgis/entrypoint.py

```diff
@@ -6,37 +6,34 @@
 from pathlib import Path
 from typing import Union
 
-import qfieldcloud.qgis.apply_deltas
-import qfieldcloud.qgis.process_projectfile
+import qfc_worker.apply_deltas
+import qfc_worker.process_projectfile
 from libqfieldsync.offline_converter import ExportType, OfflineConverter
+from libqfieldsync.offliners import OfflinerType, PythonMiniOffliner, QgisCoreOffliner
 from libqfieldsync.project import ProjectConfiguration
 from libqfieldsync.utils.file_utils import get_project_in_folder
 from libqfieldsync.utils.qgis import set_bad_layer_handler
-from qfieldcloud.qgis.utils import (
+from qfc_worker.utils import (
     Step,
     StepOutput,
     WorkDirPath,
     Workflow,
     get_layers_data,
     layers_data_to_string,
 )
-from qgis.core import (
-    QgsCoordinateTransform,
-    QgsOfflineEditing,
-    QgsProject,
-    QgsRectangle,
-    QgsVectorLayer,
-)
+from qgis.core import QgsCoordinateTransform, QgsProject, QgsRectangle, QgsVectorLayer
 
 PGSERVICE_FILE_CONTENTS = os.environ.get("PGSERVICE_FILE_CONTENTS")
 
 logger = logging.getLogger("ENTRYPNT")
```

Comment:
Eventhough unrelated to the PR i am wondering:
Why not "__name__" here? Or like the full name "docker-qgis/entrypoint.py" if it would be "__main__"?
Same for other worker related scripts where we use such names.

Reply:
Honest asnwer: I don't remember. I am afraid to change without testing, because it can break some log parsing utilities.

There is one more place where this is used: `process_project_file.py`.  We can do a follow-up PR changing this behaviour with some tests if all the rest is good.